data/mods/Xedra_Evolved/items/inventor/melee.json

```diff
@@ -62,6 +62,24 @@
     "power_draw": "350 W",
     "revert_to": "combatsaw_spear_off",
     "use_action": { "target": "combatsaw_spear_off", "msg": "You turn off the chainsaw spear.", "active": false, "type": "transform" },
-    "flags": [ "MESSY", "TRADER_AVOID", "NONCONDUCTIVE" ]
+    "flags": [ "MESSY", "TRADER_AVOID", "NONCONDUCTIVE", "INVENTOR_CRAFTED" ]
+  },
+  {
+    "id": "megachette",
+    "copy-from": "mc_broadsword",
+    "type": "TOOL",
+    "name": { "str": "megachette" },
+    "description": "A completely flat, single-edged blade.  While very visually appealing, it is no less deadly.",
+    "replace_materials": { "mc_steel": "steel", "leather": "steel" },
+    "relative": { "melee_damage": { "cut": 3 } }
+  },
+  {
+    "id": "binary_sword",
+    "copy-from": "qt_zweihander",
+    "type": "TOOL",
+    "name": { "str": "binary sword" },
+    "description": "A relatively heavy two-handed sword with \"100111100\" inscribed on the blade, perhaps to appear fancier than it truly is.  Still, it is a surprisingly good weapon",
```

Comment:
```suggestion
    "description": "A relatively heavy two-handed sword with \"100111100\" inscribed on the blade, perhaps to appear fancier than it truly is.  Still, it is a surprisingly good weapon.",
```
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/99621099/6c4f51dd-684d-4790-a612-36de5884dadf)
I understood that reference